src/OrbitGl/GpuTrack.cpp

```diff
@@ -95,7 +95,7 @@ void GpuTrack::UpdatePositionOfSubtracks() {
   submission_track_->SetHeadless(false);
   submission_track_->SetIndentationLevel(indentation_level_ + 1);
 
-  float current_y = pos[1] + layout_->GetTrackTabHeight();
+  float current_y = pos[1];
```

Comment:
I guess these two changes here result in the problem we've seen with the subtrack's labels being off (or better with the tracks not aligned to the headers).

I guess we want to add a `header_->GetHeight()` here and below, which requires a different implementation of `TrackHeader::GetHeight`.